.github/workflows/checkfornull

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
this is a simple bash script that runs grep on the playtetest output looking for "Null" which should only happen if the bot had a Null reference exception. If grep is successful, (which means it found a Null) the scripts exits with a 1 (any non-zero represents an error on the terminal). If grep fails, (which means it did not find a Null), the script exits with a 0 which is a success